From 20fff99a9a7e9ffd9a701544a6853aec1ad2be08 Mon Sep 17 00:00:00 2001
From: Ranjani Vaidyanathan <Ranjani.Vaidyanathan@freescale.com>
Date: Wed, 29 Apr 2015 10:29:27 -0500
Subject: [PATCH 1027/5242] MLK-10782-4 ARM:imx6qdl:dts:Add LDB_DI_CLK parent
 to device tree.

commit  62ae47a87290d464943bf5fbc59d132bfc15a66e from
https://source.codeaurora.org/external/imx/linux-imx.git

Add the LDB clock parents to the device tree.

Signed-off-by: Fabio Estevam <fabio.estevam@freescale.com>
Signed-off-by: Ranjani Vaidyanathan <Ranjani.Vaidyanathan@freescale.com>
(cherry picked from commit 1a6cd019c1ab62ca0dc23bbc6b033df3f15850a5)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qdl-sabreauto.dtsi |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi b/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
index b8257ae..e63b019 100644
--- a/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
@@ -396,6 +396,8 @@
 				 <&clks IMX6QDL_CLK_PLL3_USB_OTG>,
 				 <&clks IMX6QDL_CLK_PLL3_USB_OTG>;
 	assigned-clock-rates = <0>, <0>, <0>, <0>, <24576000>;
+	fsl,ldb-di0-parent = <&clks IMX6QDL_CLK_PLL2_PFD0_352M>;
+	fsl,ldb-di1-parent = <&clks IMX6QDL_CLK_PLL2_PFD0_352M>;
 };
 
 &dcic1 {
-- 
1.7.9.5

